docs/user_guide/datasets/acs_public_coverage.rst

```diff
@@ -0,0 +1,281 @@
+         1. Yes
+         2. No
+
+   *  - RAC1P
+      - Race code
+         1. White alone
+         2. Black or African American alone
+         3. American Indian alone
+         4. Alaska Native alone
+         5. American Indian and Alaska native tribes specified; or American
+            Indian or Alaska Native, not specified and no other races
+         6. Asian alone
+         7. Native Hawaiian and Other Pacific Islander alone
+         8. Some Other Race alone
+         9. Two or More races
+
+
+The target label is given by PUBCOV, which can be used for a binary
+classification task.
+
+.. list-table::
+   :header-rows: 1
+   :widths: 7 30
+   :stub-columns: 1
+
+   *  - Column name
+      - Description
+
+   *  - PUBCOV
+      - Public health coverage, with PUBCOV == 1 if the individual has public
+        health coverage, else 0
+
+
+.. _acs_public_coverage_discussion:
+
+Discussion
+^^^^^^^^^^
+
+As Ding et al. [1]_ notes, algorithmic (un)fairness must be evaluated in
+context of the intervention, and the effects of different interventions vary
+widely by state. Similarly, datasets should be carefully designed for the
+specific prediction task in mind.
+
+Therefore, to investigate how viable ACSPublicCoverage is as a fairness
+benchmark dataset, we must first ask what real-world tasks might require a
+prediction of whether an individual is covered by public health insurance.
+Usually, the entity that needs this information (e.g. the government checking
+for eligibilty in a social program) will simply ask the individual directly.
+Ding et al. do not provide any examples, so it is difficult to evaluate what
+"fair" even means without context. For now, the rest of this discussion focuses
+on preliminary data analysis of ACSPublicCoverage and identifies potential
+problems that users should be aware of beforehand.
+
+1. A uniform $30,000 threshold ignores differences between states.
+
+    Although the purpose of ACSPublicCoverage is to "predict whether a
+    low-income individual, not eligible for Medicare, has coverage from public
+    health insurance," this dataset should not be blindly used for all
+    prediction tasks that fall in that category. Ding et al. filters ACS PUMS
+    for all individuals whose income is below $30,000, but the 'low-income'
+    threshold differs for every state, so applying one threshold to all states
+    can lead to flawed results. The paper does not justify the choice of
+    $30,000, but the authors answer in `this Github issue
+    <https://github.com/zykls/folktables/issues/14>`_ that it was calculated
+    approximately in accordance with Affordable Care Act (ACA) eligibilty
+    guidelines qualifying adults with income up to 133% of the 2021 Federal
+    Poverty Line for Medicaid. Note that we examine the 2018 dataset in this
+    article, and 2018 had a different poverty line. Furthermore, not all states
+    expanded Medicaid under the ACA, so some states have a much lower poverty
```

Comment:
Is this poverty line what is meant with the "low-income threshold" mentioned earlier?

Reply:
Hmm, my understanding (from [here](https://www.census.gov/topics/income-poverty/poverty/about.html)) is that there _isn't_ a state-level poverty line, only a federal one, and it's updated annually to keep track with inflation via the CPI... but this may not be what the authors are referring to.